docs/database_schema_diagrams/source/entity_network_details.json

```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities, that is, beyond aliases, collections, edits, relationships, subscriptions, ratings, tags.",
+  "tooltip": "Tables uniquely related to primary entities",
+  "_comment": "It shows the main tables for primary entity types and tables unique to some of these, that is, beyond aliases, collections, edits, relationships, subscriptions, ratings, tags.",
```

Comment:
_overview talks about "the main tables for primary entities" while this talks about "the main tables for primary entity types". Is that intentional?

Reply:
No, it’s been addressed in the meantime.